Where_Is_My_Wife/Assets/_Main/Scripts/UI/Settings/SliderSetting.cs

```diff
@@ -33,6 +33,21 @@ public override void OnDeselect(BaseEventData eventData)
             SetHorizontalValue(0);
         }
 
+        protected override void Start()
+        {
+            base.Start();
+            
+            switch (_settingType)
+            {
+                case SettingType.Music:
+                    _slider.normalizedValue = AudioManager.MusicVolumeValue;
+                    break;
+                case SettingType.Sfx:
+                    _slider.normalizedValue = AudioManager.SfxVolumeValue;
```

Comment:
This is a bad patter cause the component is not generic, something else should load this value here.

This should be set via SettingsController so this script works generically, to achieve the save of the value  an Action here can be exposed so the Controller saves the value when changed.

DM me for more info.